src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,26 +1,43 @@
 package frc.robot;
 
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.example.*;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.IntakeConstants;
+import frc.robot.subsystems.intake.IntakeIO;
+import frc.robot.subsystems.intake.IntakeIOSim;
+import org.eclipse.jetty.util.thread.TimerScheduler;
+
+import java.util.Timer;
+
+import static frc.robot.subsystems.intake.IntakeConstants.*;
 
 public class RobotContainer {
 
     private static RobotContainer INSTANCE = null;
+    private Intake intake = Intake.getInstance();
```

Comment:
Be careful here. The `INSTANCE` inside the intake is null until the robotcontainer constructor is called. I'm not sure it's not, but better to call `getInstance` in the constructor.

Reply:
done.

Reply:
423a178e32ada435b677b785a6c731fd229beeee